### DISCUSSION TITLE
Filter using .or() and null value

### DISCUSSION THREAD

**Discussion by captainR0bbo** [supabase/supabase#4861] [Questions] [answered]
I want to query a table for values that are null or have a specific value. Specifically I have a userId column (uuid) and I want rows for the current user or that are null (system default values). 

I don't see a whole lot of examples of the .or() function. The docs show .or('column.eq.value,column.eq.secondvalue'). But column.eq.null does not work.

My code:
`
let { data: programsLocal, error } = await supabase
    .from<Program>("program")
    .select(
      "*")
    .or("userId.eq.null" + (userId ? ",userId.eq." + userId : ""));
`
Request string on my .or - 'userId.eq.null,userId.eq.240dfd6f-7fae-4096-a06b-f2f56fbbcf25'
Results in error: invalid input syntax for type uuid: "null".
Same results if just looking for null - 'userId.eq.null'

Any suggestions? Thank you!

**Comment by GaryAustin1** [comment, marked as answer]
Try .is.null

**Reply by captainR0bbo** [reply]
oh man I didn't think it would be that simple. 

Thank you so much!
